pkg/virt-operator/install-strategy/strategy.go

```diff
@@ -51,6 +51,15 @@ import (
 
 const customSCCPrivilegedAccountsType = "KubevirtCustomSCCRule"
 
+var defaultPort int32 = 443
```

Comment:
Dumb question, but is this the correct port to default to?

Reply:
Unfortunately, there's not a defined variable in the package to use so I'm going off the comment here for this value https://github.com/kubernetes/api/blob/master/admissionregistration/v1beta1/types.go#L571